docs/topics_5/web-adding-global-custom-labels.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_labels_{context}']
+= Adding global custom labels
+
+Global labels apply to all projects.
+
+The {ProductShortName} includes a preconfigured set of global labels. You can define your own custom global labels.
```

Comment:
Same comments as for rules. Move first sentence to beginning of first paragraph. Leave second sentence here.